### PR TITLE
create PURGE cmd_ to clear the cache

### PR DESCRIPTION
**Pull request by yij793**
address: https://nanopay.atlassian.net/browse/CPF-3754
issue : login with unverified account or 2FA enabled account, refresh the page, then login.
guessing: cache block getting data from menuDAO
fix: remove cacheType in menuDAO
Screen Shot 2020-01-16 at 1 58 23 PM

### CODE REVIEW

File: src/services [hunk 1]
```diff
@@ -343,7 +343,7 @@ p({
       .setOrder(new foam.core.PropertyInfo[] {foam.nanos.menu.Menu.ORDER, foam.nanos.menu.Menu.LABEL})
       .build();
   """,
-  "client": "{\"of\":\"foam.nanos.menu.Menu\",\"cacheType\":\"FULL\"}"
+  "client": "{\"of\":\"foam.nanos.menu.Menu\"}"
```
Comment (jlhughes) [inline src/services]: Can you try this with TTL.  The timeout/delay is 15s.

Comment (yij793) [reply src/services]: Changed `\"cacheType\":\"FULL\"` to `\"cacheType\":\"TTL\"`. issue still present

**Comment by jlhughes** [comment]
Solution that needs some investigation.  On successful login we want to send a 'purge' cache CMD on the menuDAO which the cachingDAO will handle.   This is all client side. 
Need to a PURGE constance to the CachingDAO and then send this on the cmd() method.  both need to be implemented. 
* add a constants: [ 'PURGE'], 
* implement cmd_(x, obj)  - test if obj equals PURGE, true clear the cache.  else pass it on. 
Then login can call dao.cmd(CachingDAO.PURGE)

File: src/foam/nanos/u2/navigation/SignIn.js [hunk 2]
```diff
@@ -85,6 +86,7 @@ foam.CLASS({
               class: 'foam.nanos.auth.ResendVerificationEmail'
             });
           } else {
+            this.menuDAO.cmd_(X, foam.dao.CachingDAO);
```
Comment (jlhughes) [inline src/foam/nanos/u2/navigation/SignIn.js]:
```suggestion
            this.menuDAO.cmd_(X, foam.dao.CachingDAO.PURGE);
```

**Pull request merged**